lambda/format-user-services/format-user-services.ts

```diff
@@ -1 +1,98 @@
-export const handler = async (): Promise<void> => {};
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import type {
+  UserRecordEvent,
+  UserServices,
+  Service,
+  TxmaEventBody,
+} from "./models";
+
+const { AWS_REGION } = process.env;
+
+export const validateSQSRecord = (record: SQSRecord): SQSRecord => {
+  console.log(`Validating the SQS record body${record.body}`);
```

Comment:
I should probably actually make this do something, woops.
Did we decide on a definition of valid?